reference-implementation/lib/transform-stream.js

```diff
@@ -134,9 +139,15 @@ function TransformStreamTransform(transformStream, chunk) {
 
   transformStream._transforming = true;
 
+  const transformer = transformStream._transformer;
   const controller = transformStream._transformStreamController;
-  const transformPromise = PromiseInvokeOrNoop(transformStream._transformer,
-                             'transform', [chunk, controller]);
+  let transformPromise;
+  if (transformer.transform === undefined) {
```

Comment:
Oh right. InvokeOrNoop already solved this: you get it, then call method.apply later if it's not undefined. But we need to either open code it or design an entirely new helper function that will only be used in exactly one caller, like `PromiseInvokeOrFallbackOrNoop` is.